dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -597,6 +598,27 @@ class ApplicationsControllerTest < ::ActionController::TestCase
       )
     end
 
+    test 'cohort view returns facilitator applications of correct statuses' do
+      expected_applications = []
+      facilitator_cohort_view_statuses = Api::V1::Pd::ApplicationsController::COHORT_VIEW_STATUSES & FACILITATOR_APPLICATION_CLASS.statuses
+      FACILITATOR_APPLICATION_CLASS.statuses.each do |status|
+        application = create FACILITATOR_APPLICATION_FACTORY, course: 'csp'
+        application.update_column(:status, status)
+        if facilitator_cohort_view_statuses.include? status
+          expected_applications << application
+        end
+      end
+
+      sign_in @workshop_admin
+      get :cohort_view, params: {role: 'csp_facilitators', regional_partner_value: 'none'}
+      assert_response :success
+
+      assert_equal(
+        expected_applications.map {|application| application[:id]}.sort,
+        JSON.parse(@response.body).map {|application| application['id']}.sort
+      )
```

Comment:
Not required, just thinking aloud: It'd be really nice if this assertion told us something about the statuses that did/did not show up when we expected them to, since that's what this test is actually checking.